components/member-card/card.module.scss

```diff
@@ -63,3 +63,31 @@
 .iconsContainer:not(:last-child) {
   margin-right: 15px;
 }
+
+.showSkills{
+  display: flex;
+  flex-wrap: wrap;
+  margin:10px 0;
+  justify-content: center;
+}
+
+.showMemberSkills{
+  display: flex;
+  flex-wrap: wrap;
+  margin:10px 0;
+  justify-content: center;
+}
```

Comment:
The same CSS is being repeated, can we just use the same class name at both places?